docs/docs/user-authentication/sso/saml.md

```diff
@@ -0,0 +1,71 @@
+---
+id: saml
+title: SAML
+---
+
+ToolJet supports SAML authentication for your workspace. The supported SAML providers are: Okta, Active Directory Federation Services, Azure AD, Auth0 and other SAML SSO providers.
+
+### Configuring SAML
+
+To enable SAML authentication, you need to configure the following workspace settings:
+
+1. Go to **Workspace Settings** > **SSO** > **SAML**.
+
+  <div style={{textAlign: 'center'}}>
+
+  <img className="screenshot-full" src="/img/sso/saml/workspaceset.png" alt="SSO :SAMP" />
+
+  </div>
+
+2. By default, SAML is disabled. Toggle it on to enable SAML authentication.
+
+  <div style={{textAlign: 'center'}}>
+
+  <img className="screenshot-full" src="/img/sso/saml/enable.png" alt="SSO :SAMP" />
+
+  </div>
+
+3. Enter the following configuration details:
+
+  - **SAML Provider Name**: Enter the name of your SAML provider. This name will be displayed on the login page.
+  - **Identity provider metadata**: Upload the data from the metadata file provided by your SAML provider. This file contains the SAML configuration details.
```

Comment:
@withshubh The part where user has to add IdP data needs bit more detail on how will the user find it and add it. Sharing the text for now, but let's try to keep this in mind in general on how we are adding information.

Updated text - 

To export the metadata as an XML file from your identity provider, initiate the download by clicking a button on your IdP dashboard. After downloading, copy the data from the XML file.

Paste the copied metadata into ToolJet's SAML SSO configuration settings. Please ensure that the metadata is pasted in the correct format, as it contains essential configuration details from the identity provider necessary for authentication.

Additionally, you can often find this data by navigating to https://<provider-domain>/federationmetadata/2007-06/federationmetadata.xml